### PR TITLE
Dashboard: add user full name and avatar.

### PR DESCRIPTION
**Pull request by Pineirin**
* closes: inveniosoftware/invenio-app-rdm#1473

Screenshot from 2022-04-28 15-00-12

### CODE REVIEW

File: invenio_app_rdm/records_ui/views/deposits.py [hunk 1]
```diff
@@ -293,11 +296,18 @@ def dashboard(dashboard_name=None):
     """Display user dashboard page."""
     if not current_app.config["COMMUNITIES_ENABLED"] or not dashboard_name:
         dashboard_name = current_app.config["_DASHBOARD_ROUTES"][0]
+    prefix = current_user_resources.users_resource.config.url_prefix
+    route = current_user_resources.users_resource.config.routes["item-avatar"]
+    image = '/api{prefix}{route}'.format(
+        prefix=prefix,
+        route=re.sub("<.+>", str(current_user.id), route)
```
Comment (ntarocco) [inline invenio_app_rdm/records_ui/views/deposits.py:303]: any better way to solve this?

Comment (ntarocco) [reply invenio_app_rdm/records_ui/views/deposits.py:303]: Another way would be to take advantage of the `links`:
```
current_user_resources.users_service.links_item_tpl.expand(current_user)['avatar']
```

```
'https://127.0.0.1:5000/api/users/3/avatar.svg'
```

File: invenio_app_rdm/records_ui/views/deposits.py [hunk 2]
```diff
@@ -293,11 +296,18 @@ def dashboard(dashboard_name=None):
     """Display user dashboard page."""
     if not current_app.config["COMMUNITIES_ENABLED"] or not dashboard_name:
         dashboard_name = current_app.config["_DASHBOARD_ROUTES"][0]
+    prefix = current_user_resources.users_resource.config.url_prefix
```
Comment (ntarocco) [inline invenio_app_rdm/records_ui/views/deposits.py:299]: can you please move this view from `deposits.py` to `records.py`? It does not belong here.

File: invenio_app_rdm/records_ui/views/deposits.py [hunk 3]
```diff
@@ -293,11 +296,18 @@ def dashboard(dashboard_name=None):
...
+        prefix=prefix,
+        route=re.sub("<.+>", str(current_user.id), route)
+    )
     return render_template(
         "invenio_app_rdm/records/dashboard.html",
         dashboard_name=dashboard_name,
         searchbar_config=dict(searchUrl=get_search_url()),
-        username=current_userprofile.full_name
+        username=current_userprofile.full_name,
+        image=image
```
Comment (kpsherva) [inline invenio_app_rdm/records_ui/views/deposits.py:310]:
```suggestion
        current_user_avatar=image
```
for better readability

File: invenio_app_rdm/records_ui/views/deposits.py [hunk 4]
```diff
@@ -293,11 +296,18 @@ def dashboard(dashboard_name=None):
...
+    image = '/api{prefix}{route}'.format(
+        prefix=prefix,
+        route=re.sub("<.+>", str(current_user.id), route)
+    )
     return render_template(
         "invenio_app_rdm/records/dashboard.html",
         dashboard_name=dashboard_name,
         searchbar_config=dict(searchUrl=get_search_url()),
-        username=current_userprofile.full_name
+        username=current_userprofile.full_name,
```
Comment (kpsherva) [inline invenio_app_rdm/records_ui/views/deposits.py:309]: if I am not mistaken `current_userprofile` might be available already in jinja, if not maybe worth having a context processor ? ping @ntarocco

File: invenio_app_rdm/records_ui/views/deposits.py (hunk 3 above)
Comment (Pineirin) [reply invenio_app_rdm/records_ui/views/deposits.py]: I did this but push didn't go through my bad.

**Review by ntarocco — APPROVED** [review]

**Review by zzacharo — APPROVED** [review]

**Pull request merged**